interceptors/logging/examples/zap/example_test.go

```diff
@@ -17,10 +17,10 @@ import (
 func InterceptorLogger(l *zap.Logger) logging.Logger {
 	return logging.LoggerFunc(func(ctx context.Context, lvl logging.Level, msg string, fields ...any) {
 		f := make([]zap.Field, 0, len(fields)/2)
+		iter := logging.Fields(fields).Iterator()
 		for i := 0; i < len(fields); i += 2 {
```

Comment:
I think it can be simplified further, similar to https://github.com/grpc-ecosystem/go-grpc-middleware/blob/main/interceptors/logging/examples/logrus/example_test.go#L21

I messed this up 🙃  Can we double check other examples too

Reply:
It actually needs to be `for iter.Next()`. This issue seems to be present on all the examples.